ext/ddtrace.stub.php

```diff
@@ -0,0 +1,496 @@
+     * @param \DateTimeInterface|int|float $startTime Start time of the span in seconds. Defaults to zero.
+     * @return SpanData The newly started span
+     */
+    function start_span(\DateTimeInterface|int|float $startTime = 0): SpanData {}
+
+    /**
+     * Close the currently active user-span on the top of the stack
+     * @param \DateTimeInterface|int|float|null $finishTime Finish time in seconds. if passing float or int, it should
+     * represent the timestamp (including as many decimal places as you need)
+     * @return void
+     */
+    function close_span(\DateTimeInterface|int|float|null $finishTime): void {}
+
+    /**
+     * Start a new trace
+     *
+     * More precisely, a new root span stack will be created and switched on to, and a new span started.
+     *
+     * @return SpanData The newly created root span
+     */
+    function start_trace_span(): SpanData {}
+
+    /**
+     * Get the active stack
+     * @return SpanStack|null A copy of the active stack, or 'null' if there is none
+     */
+    function active_stack(): SpanStack|null {}
+
+    /**
+     * Initialize a new span stack and switch to it. If tracing isn't enabled, a root span stack will be created.
+     * @return SpanStack The newly created span stack
+     */
+    function create_stack(): SpanStack {}
+
+    /**
+     * Switch back to a specific stack (even if there is no active span on that stack), or to the parent of the active
+     * stack if no stack is given.
+     * @param SpanData|SpanStack|null $newStack Stack to switch to. If 'null' is given, switches to the parent of the
+     * active stack. If a SpanData object is given, it will switch to the stack of the latter.
+     * @return SpanStack|null The newly active stack, or 'null' if there wasn't an active stack and no specific stack
+     * was given as an argument to the function call
+     */
+    function switch_stack(SpanData|SpanStack|null $newStack = null): null|SpanStack {}
+
+    /**
+     * Set the priority sampling level
+     * @param int $priority The priority level to be set to
+     * @param bool|null $global If set to 'true' and if there is no active stack (or the active stack doesn't have a
+     * root span), then the default priority sampling will be set to the provided priority level. Otherwise, the root's
+     * priority sampling level will be updated with the new value.
+     * @return void
+     */
+    function set_priority_sampling(int $priority, ?bool $global = false): void {}
+
+    /**
+     * Get the priority sampling level
+     * @param bool|null $global If set to 'true' and if there is no active stack (or the active stack doesn't have a
+     * root span), then the default priority sampling will be returned, else it will be fetched from the root.
+     * @return int The priority sampling level
+     */
+    function get_priority_sampling(?bool $global = false): int {}
+
+    /**
+     * Sanitize an exception
+     * @internal
```

Comment:
```suggestion
```
Hm, let's not mark that as internal. It's an utility function everyone should be able to use